### DISCUSSION TITLE
两个建议

### DISCUSSION THREAD

**Discussion by ghost** [official-pikafish/Pikafish#14] [General]
一、用Rust语言开发皮卡鱼，用Rust重新设计。
二、提供安卓和iOS的安装包、Mac OS的安装包、GNU/Linux的软件包。

**Comment by PikaCat-OuO** [comment]
Pikafish focus in C++ only, so as Stockfish. If you like to go for rust implementation, you can always do it yourself, as I know that huynq55(https://github.com/huynq55) already have the plan to do it.
For IOS, Android and MacOS, consider using chessroad-lite(https://github.com/hezhaoyun/chessroad-lite).
For linux and arm, we already contain binaries in the releases.
Also, https://xiangqiai.com is a very good option for all platforms.
